data-migration/migrate-mysql-tidb-above-tb.md

```diff
@@ -0,0 +1,273 @@
+
+2. View the `metadata` file in the `${data-path}` directory. This is a Dumpling-generated metadata file. Record the binlog position information, which is required for the incremental replication in Step 3.
+
+    ```
+    SHOW MASTER STATUS:
+    Log: mysql-bin.000004
+    Pos: 109227
+    GTID:
+    ```
+
+## Step 2. Import full data to TiDB
+
+1. Create the `tidb-lightning.toml` configuration file:
+
+    {{< copyable "" >}}
+
+    ```toml
+    [lightning]
+    # log.
+    level = "info"
+    file = "tidb-lightning.log"
+
+    [tikv-importer]
+    # "local": Default backend. The local backend is used to import large volumes of data (1 TiB or more). During the import, the target TiDB cluster cannot provide any service.
+    # "tidb": The "tidb" backend is used to import data less than 1 TiB. During the import, the target TiDB cluster can provide service normally. For more information on the backends, refer to https://docs.pingcap.com/tidb/stable/tidb-lightning-backends.
+    backend = "local"
+    # For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage and exclusive I/O for the directory.
+    sorted-kv-dir = "${sorted-kv-dir}"
+
+    [mydumper]
+    # The data source directory. The same directory where Dumpling exports data in Step 1.
+    data-source-dir = "${data-path}"
+
+    # Configures the wildcard rule. The default rule filters out all tables in these system databases: mysql, sys, INFORMATION_SCHEMA, PERFORMANCE_SCHEMA, METRICS_SCHEMA, INSPECTION_SCHEMA.
+    # If you do not configure this item, TiDB Lightning reports a `cannot find schema` exception when it imports system tables.
+    filter = ['*.*', '!mysql.*', '!sys.*', '!INFORMATION_SCHEMA.*', '!PERFORMANCE_SCHEMA.*', '!METRICS_SCHEMA.*', '!INSPECTION_SCHEMA.*']
+
+    [tidb]
+    # The target TiDB cluster information.
+    host = ${host}                # e.g.: 172.16.32.1
+    port = ${port}                # e.g.: 4000
+    user = "${user_name}"         # e.g.: "root"
+    password = "${password}"      # e.g.: "rootroot"
+    status-port = ${status-port}  # Obtains the table schema information from TiDB status port, e.g.: 10080
+    pd-addr = "${ip}:${port}"     # The cluster PD address, e.g.: 172.16.31.3:2379. TiDB Lightning obtains some information from PD. When backend = "local", you must specify status-port and pd-addr correctly. Otherwise, the import will be abnormal.
+    ```
+
+    For more information on TiDB Lightning configuration, refer to [TiDB Lightning Configuration](/tidb-lightning/tidb-lightning-configuration.md).
+
+2. Start the import by running `tidb-lightning`. If you launch the program directly in the command line, the program might exit because of the `SIGHUP` signal. In this case, it is recommended to run the program using a `nohup` or `screen` tool. For example:
```

Comment:
`the program might exit because of the SIGHUP signal` => `the process might exit unexpectedly after receiving a SIGHUP signal`